src/lxc/tools/lxc_attach.c

```diff
@@ -286,10 +284,10 @@ static int get_pty_on_host(struct lxc_container *c, struct wrapargs *wrap, int *
 	int ret = -1;
 	struct wrapargs *args = wrap;
 
-	INFO("Trying to allocate a pty on the host");
+	printf("Trying to allocate a pty on the host\n");
```

Comment:
Ah, this is something I've forgotten to mention. All log functions - excluding  `SYSERROR()` never print to `std{out,err}` only to the log of the container. So if you see such functions that just provide basically debug or informational information should not be retained since they would spam the user with useless message on std{out,err}. Basically, if it is not an error condition it should probably be removed. So just remove this `printf()` here. The other `fprintf()`s look good to me. :) Thanks, good start!